### PR TITLE
Fix panics and action names

### PR DESCRIPTION
**Pull request by dirkkul**
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: usecases/auth/authorization/rbac/types.go [hunk 1]
```diff
@@ -52,10 +52,21 @@ func policy(permission *models.Permission) (resource, verb, domain string) {
 	case cluster:
 		resource = authorization.Cluster()
 	case collections:
+		if permission.Collection == nil {
+			return
```
Comment (moogacs) [inline usecases/auth/authorization/rbac/types.go:56]: **issue:** that should never happen `permission.Collection` default is "*" , we need to check against and make the default for resource '*'

File: usecases/auth/authorization/rbac/types.go [hunk 2]
```diff
@@ -64,28 +75,42 @@ func policy(permission *models.Permission) (resource, verb, domain string) {
...
+	} else if domain == "tenants" {
+		action += "_tenants"
+	}
+
 	action = strings.ReplaceAll(action, "_*", "")
 	permission := &models.Permission{
 		Action: &action,
 	}
 
+	noResource := policy[1] == ""
```
Comment (moogacs) [inline usecases/auth/authorization/rbac/types.go:89]: **issue (blocking):** that shall never happen

File: usecases/auth/authorization/rbac/types.go [hunk 3]
```diff
@@ -64,28 +75,42 @@ func policy(permission *models.Permission) (resource, verb, domain string) {
...
+		case objects:
+			permission.Object = &splits[4]
+		case rolesD:
+			// permission.Roles = &splits[4]
+		// case cluster:
+
+		case "*":
+			permission.Collection = &all
+			permission.Tenant = &all
+			permission.Object = &all
```
Comment (moogacs) [inline usecases/auth/authorization/rbac/types.go:112]: **suggestion:** both paths missing `permission.Role`

Comment (dirkkul) [reply usecases/auth/authorization/rbac/types.go]: fixed, rebased onto feat/rbac that included this

**Review by moogacs — APPROVED** [review]

**Pull request merged**